src/Services/RabbitMQService.cs

```diff
@@ -17,22 +20,24 @@ internal sealed class RabbitMQService : IRabbitMQService
         private readonly string _userName;
         private readonly string _password;
         private readonly int _port;
+        private readonly SslPolicyErrors _acceptablePolicyErrors;
 
-        public RabbitMQService(string connectionString, string hostName, string userName, string password, int port)
+        public RabbitMQService(string connectionString, string hostName, string userName, string password, int port, SslPolicyErrors acceptablePolicyErrors)
         {
             _connectionString = connectionString;
             _hostName = hostName;
             _userName = userName;
             _password = password;
             _port = port;
+            _acceptablePolicyErrors = acceptablePolicyErrors;
 
-            ConnectionFactory connectionFactory = GetConnectionFactory(_connectionString, _hostName, _userName, _password, _port);
+            ConnectionFactory connectionFactory = GetConnectionFactory(_connectionString, _hostName, _userName, _password, _port, _acceptablePolicyErrors);
 
-            _model = connectionFactory.CreateConnection().CreateModel();
+            _model = connectionFactory.CreateConnection(Assembly.GetEntryAssembly().GetName().Name).CreateModel();

```

Comment:
When creating the connection we pass through the Assembly name so the connection can be identified in the RabbitMQ management portal